package.json

```diff
@@ -2,46 +2,50 @@
   "name": "gulp-htmlmin",
   "description": "gulp plugin to minify HTML.",
   "version": "1.3.0",
-  "author": {
-    "name": "Jon Schlinkert",
-    "url": "https://github.com/jonschlinkert"
-  },
+  "homepage": "https://github.com/jonschlinkert/gulp-htmlmin",
+  "author": "Jon Schlinkert (https://github.com/jonschlinkert)",
   "contributors": [
     "Shinnosuke Watanabe <snnskwtnb@gmail.com> (https://github.com/shinnn)"
   ],
   "repository": "jonschlinkert/gulp-htmlmin",
+  "bugs": {
+    "url": "https://github.com/jonschlinkert/expand-files/issues"
+  },
   "license": "MIT",
-  "keywords": [
-    "format",
-    "gulp",
-    "gulpplugin",
-    "htm",
-    "html",
-    "minifier",
-    "minification",
-    "minify"
-  ],
   "files": [
     "index.js"
   ],
+  "main": "index.js",
```

Comment:
> Currently the script does not work (for me at least)

It works. https://travis-ci.org/jonschlinkert/gulp-htmlmin/builds/87939547#L197

The only reason why currently it doesn't work is that you've removed it from `devDependencies`. https://github.com/jonschlinkert/gulp-htmlmin/commit/1681ac0c885f3dfb5e71dca2f0c9cb985c57c433#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L42 `eslint` is (of course) not installed on Travis CI container by default.